1-exercises/H-array-methods-2/exercise.js

```diff
@@ -15,8 +15,8 @@ let everyone = [
   "Swathi",
 ];
 
-let firstFive; // complete this statement
-let lastFive; // complete this statement
+let firstFive = everyone.slice(0,5); // complete this statement
+let lastFive = everyone.slice(2 , 7); // complete this statement
```

Comment:
-5 is the correct parameter, since the number of elements in the array may change in the future.